install-cni.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
can you please make sure volume mount donot require any permission.if require please try to fix that.Ex: as observed in openshift selinux issue.

Reply:
will do this in next PR